FlowCrypt/Functionality/Services/IdTokenUtils.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  IdTokenUtils.swift
+//  FlowCrypt
+//
+//  Created by Ioan Moldovan on 3/25/22
+//  Copyright © 2017-present FlowCrypt a. s. All rights reserved.
+//
+
+import Foundation
+
+class IdTokenUtils {
+    // Get id token from user or user email
+    static func getIdToken(user: User? = nil, userEmail: String? = nil) async throws -> String {
+        if let user = user, case .oAuthGmail = user.authType {
+            return Imap(user: user).imapSess?.oAuth2Token ?? ""
+        }
```

Comment:
This is wrong. Imap should be used for password auth type.

Reply:
Oops. Thanks for your notice.
Just fixed it.